ansible/Dockerfile

```diff
@@ -1,5 +1,23 @@
 FROM alpine:3.6
-RUN apk update
-RUN apk add git \
-    ansible
-RUN echo "'Done !"
\ No newline at end of file
+
+ARG CI_HELPER_VERSION
+
+WORKDIR /root
+RUN apk update &&\
+ apk add -q --no-cache 
+RUN apk add bash 
```

Comment:
To get an image that doesn't need 100 layers, it's a good practice to chain all the related commands in a single `RUN`:
```docker
RUN echo "Install Ansible" && \
    apk update && \
    apk add -q --no-cache bash ca-certificates curl git openssh-client python py-pip ansible && \
    echo "Done install Ansible" && \
    echo "Install CI Helper" && \
…
```